python-sdk/telephony-sip.md

```diff
@@ -19,6 +21,23 @@ The Python SDK provides the following capabilities:
 
 - [Subscribing to Events (transcript, questions, action-items, etc.)](#subscribe-to-events)<br/>
 
+## Credential Handling 
+
+Symbl.ai's Python SDK simplifies the credential handling by allowing you either to add your credentials directly to the connection method's calls or else through a separate file saved to your execution directory. 
```

Comment:
Check grammar: "by allowing you either to add your" change to "by allowing you to either add your". 
Please make these changes to the PSTN page as well.